renovate.json5

```diff
@@ -1,6 +1,6 @@
 {
     '$schema': 'https://docs.renovatebot.com/renovate-schema.json',
-    'extends': ['github>Lombiq/renovate-config:default.json5'],
+    'extends': ['github>Lombiq/renovate-config:default.json5']
```

Comment:
This makes the JSON invalid (and trailing commas on every line are a good practice anyway, and for similar cases we also have ESLint and C# analyzer rules).

Reply:
Ah it was just a leftover.